src/tool/csv2yaml.cpp

```diff
@@ -3451,3 +3463,20 @@ static bool mob_parse_row_chatdb(char* fields[], int columns, int current) {
 	return true;
 }
 
+
+// Copied and adjusted from int_guild.cpp
+static bool exp_guild_parse_row(char* split[], int column, int current) {
+	t_exp exp = strtoull(split[0], nullptr, 10);
+
+	if (exp > MAX_GUILD_EXP) {
+		ShowError("exp_guild: Invalid exp %" PRIu64 " at line %d, exceeds max of %" PRIu64 "\n", exp, current, MAX_GUILD_EXP);
+		return false;
+	}
+
+	body << YAML::BeginMap;
+	body << YAML::Key << "Level" << YAML::Value << (current+1);
```

Comment:
Isnt "current" the line number here?
Or does it ignore the empty lines / comments here?

Reply:
It seems 'current' ignores the empty lines and comments